satellite/console/consoleweb/server.go

```diff
@@ -320,19 +359,24 @@ func (server *Server) cancelPasswordRecoveryHandler(w http.ResponseWriter, r *ht
 	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
 	recoveryToken := r.URL.Query().Get("token")
-	if len(recoveryToken) == 0 {
-		http.Redirect(w, r, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
-	}
 
 	// No need to check error as we anyway redirect user to support page
 	_ = server.service.RevokeResetPasswordToken(ctx, recoveryToken)
 
+	// TODO: Should place this link to config
 	http.Redirect(w, r, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
```

Comment:
Yes, use a config flag.

Reply:
want to do it in separate pr, because it will change our deployment flow and this pr could ne not approved some time